setup.py

```diff
@@ -60,22 +60,29 @@ def run(self):
     name            = name,
     version         = version_ns['__version__'],
     packages        = packages,
-    description     = "Jupyter protocol implementation and client libraries",
+    description     = 'Jupyter protocol implementation and client libraries',
     author          = 'Jupyter Development Team',
     author_email    = 'jupyter@googlegroups.com',
     url             = 'https://jupyter.org',
     license         = 'BSD',
     platforms       = "Linux, Mac OS X, Windows",
     keywords        = ['Interactive', 'Interpreter', 'Shell', 'Web'],
+    project_urls    = {
+        'Documentation': 'https://jupyter-client.readthedocs.io',
+        'Source': 'https://github.com/jupyter/jupyter_client/',
+        'Tracker': 'https://github.com/jupyter/jupyter_client/issues',
+    },
     classifiers     = [
+        'Framework :: Jupyter',
         'Intended Audience :: Developers',
+        'Intended Audience :: Education',
         'Intended Audience :: System Administrators',
         'Intended Audience :: Science/Research',
         'License :: OSI Approved :: BSD License',
+        'Operating System :: Independent',
```

Comment:
The spelling is `Operating System :: OS Independent` . Tautological, I know, but I think PyPI refuses uploads if the classifiers don't match [this list](https://pypi.org/pypi?%3Aaction=list_classifiers).

Reply:
Good catch. Fixing.